common/util.go

```diff
@@ -23,3 +29,19 @@ func DoStop(stop *chan bool) bool {
 		return false
 	}
 }
+
+// RelPath returns a string containing a absolute to the provided path, relative to the caller directory
```

Comment:
comment on exported function RelPathToAbs should be of the form "RelPathToAbs ..."